app/src/main/java/com/github/orkest/Model/Profile.kt

```diff
@@ -13,9 +13,9 @@ package com.github.orkest.Model
  *
  */
 data class Profile (var username: String,
-                    var profilePictureId: Int,
-                    var bio: String,
-                    var nbFollowers: Int,
-                    var nbFollowing: Int,
-                    var favoriteSongs: List<Song>,
-                    var sharedMusic: List<Song>)
\ No newline at end of file
+                    var profilePictureId: Int = -1,
```

Comment:
Is the profile picture Id never a negative value ?

Reply:
Assuming the ID will most likely be the index associated to the image in the database, I put -1 as referring to a profile with no pictureID